### PR TITLE
Use the task-manager stomp server in order to subscribe to only displ…

### PR DESCRIPTION
**Pull request by titus6012000**
…ayed tasks updates

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
Yes

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
new feature

**What is the current behavior?** *(You can also link to an open issue here)*
When an annual file is imported, the front end received via his websocket all the timestamp update even if timestamps are not displayed

**What is the new behavior (if this is a feature change)?**
we received only updates for elements displayed

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/components/global-view-core.js [hunk 1]
```diff
@@ -238,6 +248,7 @@ const GlobalViewCore = ({ timestampMin, timestampMax, timestampStep }) => {
 
     return (
         <div>
+            {createWS()}
```
Comment (TheMaskedTurtle) [inline src/components/global-view-core.js:251]: Can you create the SockJsClient directly and define method that returns only the list of topics ?

File: src/components/process-timestamp-view.js [hunk 2]
```diff
@@ -46,39 +46,48 @@ const ProcessTimestampView = ({
...
-        <Grid container direction="column">
-            <Grid item>
-                <TableHeader
-                    taskData={timestampData}
-                    processName={processName}
-                    timestamp={timestamp}
-                    onTimestampChange={onTimestampChange}
-                />
+        <div>
+            {createWS(timestamp)}
```
Comment (TheMaskedTurtle) [inline src/components/process-timestamp-view.js:74]: Same

File: package.json [hunk 3]
```diff
@@ -64,7 +65,6 @@
     "devDependencies": {
         "prettier": "2.0.5",
         "eslint-config-prettier": "^2.9.0",
-        "eslint-plugin-prettier": "^2.6.0",
-        "react-confirm-alert": "^3.0.5"
```
Comment (TheMaskedTurtle) [inline package.json:68]: Isn't it useful ?

File: src/components/process-timestamp-view.js [hunk 4]
```diff
@@ -46,39 +46,48 @@ const ProcessTimestampView = ({
 
     const handleTimestampMessage = useCallback(
         async (event) => {
-            const data = JSON.parse(event.data);
+            const data = event;
```
Comment (TheMaskedTurtle) [inline src/components/process-timestamp-view.js:49]: So you can directly use event, isn't it ?

File: src/index.js [hunk 5]
```diff
@@ -12,7 +12,6 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import './index.css';
-import 'react-confirm-alert/src/react-confirm-alert.css';
```
Comment (TheMaskedTurtle) [inline src/index.js:15]: Isn't it useful ?

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]
You can also remove upload-button.js file and maybe not remove the package-lock.json file

File: package.json (hunk 3 above)
Comment (titus6012000) [reply package.json:68]: Totally useless. this dependency has been abandoned

File: src/components/process-timestamp-view.js (hunk 4 above)
Comment (titus6012000) [reply src/components/process-timestamp-view.js:49]: Yes, the parsing is done by the library

File: src/index.js (hunk 5 above)
Comment (titus6012000) [reply src/index.js:15]: useless, replace by a component from materail-ui

File: src/components/process-timestamp-view.js (hunk 2 above)
Comment (titus6012000) [reply src/components/process-timestamp-view.js:74]: same

File: src/components/global-view-core.js (hunk 1 above)
Comment (titus6012000) [reply src/components/global-view-core.js]: The createWS function has only two instructions, which could be written in two lines without the very very strict check-style.
I will split the websocket functionality across the file

File: src/components/process-timestamp-view.js (hunk 4 above)
Comment (TheMaskedTurtle) [reply src/components/process-timestamp-view.js:49]: I mean, no need to define `const data = event` and use directly `event` as a variable

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: src/components/process-timestamp-view.js (hunk 4 above)
Comment (titus6012000) [reply src/components/process-timestamp-view.js]: indeed, I wanted to do the minimum changes

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**